pkg/ctl/update/cluster.go

```diff
@@ -100,10 +100,27 @@ func doUpdateClusterCmd(cmd *cmdutils.Cmd) error {
 
 	stackManager := ctl.NewStackManager(cfg)
 
+	if versionUpdateRequired {
+		msgNodeGroupsAndAddons := "you will need to follow the upgrade procedure for all of nodegroups and add-ons"
+		cmdutils.LogIntendedAction(cmd.Plan, "upgrade cluster %q control plane from current version %q to %q", cfg.Metadata.Name, currentVersion, cfg.Metadata.Version)
```

Comment:
Interesting: there are people reporting that eksctl upgrades to the next version regardless of what they specify in the config file. But here it mentions that it will upgrade to what they want. https://github.com/weaveworks/eksctl/issues/915

Reply:
That bug report is correct. It does upgrade to the next version without looking at the version in the config file.

Reply:
Thanks. Let's discuss that as a separate issue